wherehows-web/.eslintrc.js

```diff
@@ -2,42 +2,27 @@ module.exports = {
   root: true,
   globals: {
     server: true,
-    define: true,
+    define: true
```

Comment:
doesn't prettier enforce trailing comma? :p

Reply:
Prettier removes trailing commas by default, but you can change that option if you want

https://prettier.io/docs/en/options.html#trailing-commas

Reply:
@justYves